### PR TITLE
Refactored the "near" method in a custom find

### PR DESCRIPTION
**Pull request by real34**
For retrocompatibility, the near() method now uses the custom find.
The function find in GeoAddress has no sense anymore and has been removed.

### CODE REVIEW

**Comment by mariano** [comment]
Ok I know what you are aiming but I'm not sure I'm totally comfortable with removing a whole set of tests (testFind()). If you are creating a 'near' custom find type, why the need to remove those tests?

There's also the need to make this CakePHP 2.0 compatible, at least from the code. Like the usage for Model::$_findMethods should check if instead it should be using Model::$findMethods (CakePHP 2.0). Similarly, at one point you are using Model::_findCount(), which is protected in 2.0 and thus would trigger a PHP warning.

The bottom line is I like the idea but needs to be ironed out a bit. I'll see if I can find the time (been crazy lately), but if you wanna take a go please feel free. Its awesome that you are helping me out :)

**Comment by real34** [comment]
Just a quick update to let you know that this small refactoring is on my todo-list ... I just cannot find the time to work on it right now. Sorry for that, it might have to wait a few more weeks!
